packages/server/src/choiceCardAmounts.ts

```diff
@@ -0,0 +1,18 @@
+import { cacheAsync } from './lib/cache';
+import { isProd } from './lib/env';
+import { fetchS3Data } from './utils/S3';
+import { ChoiceCardAmounts } from '@sdc/shared/dist/types';
+
+const getChoiceCardAmounts = (): Promise<ChoiceCardAmounts> =>
+    fetchS3Data('support-admin-console', `${isProd ? 'PROD' : 'CODE'}/amounts.json`).then(
```

Comment:
Hey @michaelbjacobson this is the old amounts file. The one that gets updated by the admin console now is called `configured-amounts.json` - the model is a little different as it allows for ab tests in the different currency regions

Reply:
This is the model: https://github.com/guardian/support-admin-console/blob/main/app/models/Amounts.scala#L20

Reply:
Ah cool, do you know where the model for this is defined? I can reverse engineer it if not.

Reply:
That link ^^ is for the scala version of the model. We also decode that data in support-frontend so have a flow implementation: https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/contributions.js#L71

Reply:
here's the TS model https://github.com/guardian/support-admin-console/blob/3939e1e38262cfc36719610e0e874e00a5008ce4/public/src/components/amounts/configuredAmountsEditor.tsx#L42

Reply:
ah yes, it's also in support-frontend!

Reply:
Ahhhh ofc! I don't know how I completely blanked on that one 😂

Reply:
✅